catalog/ui/catalog-ui-search/src/main/webapp/js/LessVariables.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global require*/
+var lessToJs = require('less-vars-to-js');
+var backbone = require('backbone');
+var $ = require('jquery');
+
+var Backbone = require('backbone');
+
+module.exports = new (Backbone.Model.extend({
+    defaults: {
+        initialized: false,
+        '@minimumSpacing': '0.625rem',    
+        '@minimumScreenSize': '20rem',
+        '@minimumButtonSize': '2.75rem'
+    },
+    initialize: function(){
+        $.get({
```

Comment:
I looked at trying something like that first, if you have some time can you try for me?  I had issues where it would give me the style loader code instead, so it seems to be clashing with it.  I really would like to avoid the request if we can.